docs/faq.rst

```diff
@@ -4,6 +4,19 @@
 Frequently Asked Questions
 ==========================
 
+Which languages does the Rasa NLU support?
+------------------------------------------
+Rasa NLU can be used to understand any language, but some backends are restricted to specific languages.
+
+The ``tensorflow_embedding`` pipeline can be used for any language, because
```

Comment:
mention here that you do need a tokenizer which works. For some languages whitespace tokenizers work, for e.g. japanese and chinese you need a custom one